packages/synapse-react-client/src/utils/hooks/useUploadFileEntity/useCreatePathsAndGetParentId.ts

```diff
@@ -9,24 +9,24 @@ export function useCreatePathsAndGetParentId() {
   const { mutateAsync: createFolderPath } = useCreateFolderPath()
 
   return useMutation({
-    mutationFn: (args: { file: File; parentId: string }) => {
-      const { file, parentId } = args
+    mutationFn: (args: { file: File; rootContainerId: string }) => {
+      const { file, rootContainerId } = args
```

Comment:
This function has a return type `{ file: File; parentId: string }`, so I changed the argument type to `{ file: File; rootContainerId: string }` to make it more clear that the `parentId` may be different from the `rootContainerId`